src/poetry/puzzle/provider.py

```diff
@@ -666,23 +666,23 @@ def complete_package(
         )
 
         # Searching for duplicate dependencies
-        #
+
```

Comment:
Is there a rule that removes these empty comment lines? I tried myself and it seems pre-commit succeeds.

I'd prefer to keep them.

Reply:
That was me incorrectly restoring from a change to docstrings. Brought those back